package.json

```
@@ -37,6 +37,7 @@
         "@fortawesome/react-fontawesome": "0.1.9",
         "axios": "0.19.2",
         "hover.css": "2.3.2",
+        "html-react-parser": "^0.14.0",
```

```suggestion
        "html-react-parser": "0.14.0",
```

Make the change then delete your `yarn.lock` file and your `node_modules` folder and rerun `yarn install` then push the new `yarn.lock` there should not be any `^` or `~` symbols before a version in this file.